main.go

```diff
@@ -44,7 +44,7 @@ Options:
         -o          List only already-vendored dependencies.
     -C --clean      Detect all unused vendored dependencies and remove it.
     -T --tree       Show dependencies tree.
-	  -i --import   Show used import path instead of git repo.
```

Comment:
there is an indentation because `-i --import` is subkey only for `-T --tree`, like `-o` for `-Q --query`

Reply:
I reverted this change.  Maybe you could put some extra spaces in to line up the explanatory text, eg.

...
    -T --tree    ** add extra space here to line up columns **   Show dependencies tree.
	  -i --import   Show used import path instead of git repo.